### PR TITLE
fix spatial alignment key_added

### PR DESCRIPTION
**Pull request by ArinaDanilina**

### CODE REVIEW

File: examples/annotation_mapping.ipynb [hunk 1]
```diff
@@ -0,0 +1,313 @@
+{
```
Comment (MUCDK) [inline examples/annotation_mapping.ipynb:1]: <p>ignore the error messages </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (MUCDK) [inline examples/annotation_mapping.ipynb:1]: <p>ignore commments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Comment by ArinaDanilina** [comment]
<p>the dataset description appears on mouseover so i only clarified the difference in spatial or not</p>

---

 <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/57/discussion/#comment-1907937358'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by MUCDK — APPROVED** [review]
Thanks Arina!

**Pull request merged**